libyaml/src/Text/Libyaml.hs

```diff
@@ -663,18 +682,59 @@ parserParseOne' parser = allocaBytes eventSize $ \er -> do
           return $ Left $ YamlParseException problem context problemMark
         else Right <$> getEvent er
 
+-- | Whether a tag should be rendered explicitly in the output or left
+-- implicit.
+--
+-- @since 0.11.1.0
+data TagRender = Explicit | Implicit
+  deriving (Enum)
+
+toImplicitParam :: TagRender -> CInt
+toImplicitParam Explicit = 0
+toImplicitParam _ = 1
```

Comment:
Minor comment, but I'd prefer to avoid wildcards and be explicit in the constructor names.